recipes/master.rb

```diff
@@ -87,6 +98,28 @@
   end
 end
 
+# salt-api default user acl
+# TODO: to be replaced by LWRP `external_auth`
+default_user_acl = {
+  'external_auth' => {
+    'pam' => {
+      'saltapi' => [
+        '.*',
+        '@wheel',
+        '@runner',
+      ],
+    },
+  },
+}.to_yaml
+
+file 'default-api-user.conf' do
```

Comment:
do you mean `file[default-api-user.conf` user? I will also add a check to remove this file if `api` is not enabled.